src/AppInstallerCommonCore/Public/winget/Fonts.h

```diff
@@ -81,6 +81,7 @@ namespace AppInstaller::Fonts
         FontStatus Status = FontStatus::Unknown;
         std::wstring PackageId;
         std::wstring PackageVersion;
+        std::wstring PackageIdentifier;
```

Comment:
nit: Having both an `Id` and an `Identifier` is somewhat confusing.  Doesn't need to change for merge, but maybe think about renaming one of them in the future.

Reply:
Next time I touch it I'll change it to "UniqueIdentifier" or something similar away from the Package similarity.